lib/kele.rb

```diff
@@ -0,0 +1,16 @@
+require 'httparty'
+
+class Kele
+  include HTTParty
+
+
+  def initialize(email,password)
+    @baseURL = 'https://www.bloc.io/api/v1/'
+    @email = email
```

Comment:
Will you ever need to access email and password outside of initialize? If not, consider making these regular variables.